src/Database/Driver.php

```diff
@@ -63,7 +63,7 @@ public function __construct($config = [])
         $config += $this->_baseConfig;
         $this->_config = $config;
         if (!empty($config['quoteIdentifiers'])) {
-            $this->autoQuoting(true);
+            $this->enableAutoQuoting(true);
```

Comment:
You can add it on top, the commit itself will show the differences.
With a method called enableX() it is kind of obvious that this should be enabling by default.
It might be different with setX() methods, but here it is sure true :)